phpstan.neon

```diff
@@ -0,0 +1,9 @@
+parameters:
+    checkGenericClassInNonGenericObjectType: false
+    checkMissingIterableValueType: false
+    excludePaths:
+    ignoreErrors:
+        - '#Parameter \$.+ of method .+ has invalid typehint type.+#'
+        - '#Return typehint of method .+ has invalid type.+#'
+        - '#.+Transfer.+#'
```

Comment:
I guess you should change this ignore pattern to more specifics. It is too common.